index.html

```diff
@@ -7,57 +7,215 @@
             <title>Sandeep's Portfolio</title>
             <link rel="stylesheet" type="text/css" href="style.css">
         </head>
-        <body>
-            <!-- navbar section -->
+    <body>
+        <!-- navbar section -->
             <section>
-                <header class="navbar">
-                    <a href="#">
-                        <h4>Sandeep</h4>
-                    </a>
-                    <nav>
-                        <div id="bar">
-                            <img src="images/Menu.png" alt="icon">
-                            <img src="images/Menu.png" alt="icon">
-                            <img src="images/Menu.png" alt="icon">
-                        </div>
-                    </nav>
-                </header>
+                    <header class="navbar">
+                        <a href="#">
+                            <h4>Sandeep</h4>
+                        </a>
+                        <nav>
+                            <div id="bar">
+                                <img src="images/Menu.png" alt="icon">
+                                <img src="images/Menu.png" alt="icon">
+                                <img src="images/Menu.png" alt="icon">
+                            </div>
+                        </nav>
+                    </header>
             </section>
-            <!-- headline section -->
+                <!-- headline section -->
             <section>
-                <div class="headline">
-                    <h1 class="h1">Hey There, I'm Sandeep..</h1>
```

Comment:
- [ ] [OPTIONAL] Kindly ensure that you follow suit with all the design guidelines to achieve an optimized pixel-perfect design like you'll be doing in your next job. Note that everything must much as possible, the font size, font weight, margin, padding, background image, color, line height, font family, and the like.

- Your design
<img width="131" alt="wrong" src="https://user-images.githubusercontent.com/41163571/227705732-5219ef5f-be84-46d9-a1f3-1bb0468369e7.png">

- Figma design
<img width="217" alt="right" src="https://user-images.githubusercontent.com/41163571/227010023-9fcfb014-309c-4293-ba46-014f8e1c0000.png">

I took the extra mile for your sake to confirm if the font existed and was for free, guess what? 
[it is!!!!!!!](https://fonts.google.com/specimen/Crete+Round?query=crete).

- You can always get the properties on Figma by clicking on the item, and then check the right side pane for the information.

Reply:
Fixed the font as highlighted.